modules/dataquery/jsx/react.app.js

```diff
@@ -440,6 +440,26 @@ var DataQueryApp = React.createClass({
                 loading: false
             }
         });
+        for (var i = 0; i < fieldsList.length; i++) {
+            var that = this;
+            $.ajax({
+                url: loris.BaseURL + "/AjaxHelper.php?Module=dataquery&script=datadictionary.php",
+                success: function(data) {
```

Comment:
If you use an arrow function here, i.e. ```(data) => { ... } ``` you can remove the above `var that = this;`. Arrow function don't bind their own `this`.